driver/driver.hpp

```diff
@@ -151,7 +151,9 @@ inline void PadBufferSize(size_t& sz, int datatype_sz)
            "pool[fp16], lrn[fp16], "
            "activ[fp16], softmax[fp16], bnorm[fp16], rnn[fp16], gemm[fp16], ctc, dropout[fp16], "
            "tensorop[fp16], reduce[fp16|fp64], layernorm[bfp16|fp16], sum[bfp16|fp16], "
-           "argmax[bfp16|fp16], groupnorm[bfp16|fp16], cat[bfp16|fp16]\n");
+           "argmax[bfp16|fp16], groupnorm[bfp16|fp16], cat[bfp16|fp16], "
+           "image_adjust_hue[_fp16|_bfp16], image_adjust_saturation[_fp16|_bfp16], "
+           "image_normalize[_fp16|_bfp16] \n");
```

Comment:
I think you should follow the naming convention and remove the "_".
There're also existing some long naming op in AMD code base https://github.com/ROCm/MIOpen/blob/develop/driver/driver.hpp

Reply:
I've removed the underscores